### ISSUE TITLE
Allow viewing properties such as file path if imported from cozy-files

### ISSUE THREAD

**Issue by nebajoth** [cozy-labs/cozy-music#55]
I've freshly installed cozy-music from the cozy app store. Cozy-music shows duplicates of every track that I imported from cozy-files. It may just be that it synced them automatically when I installed the app from the store, and that I then re-ran it manually. In my case, since the file exists only once in cozy-files, I can delete either one and have it not matter. But a really good sanity check like being able to glance at the path to the referenced file would be appreciated, and natural to have. I could see REALLY wanting it when I have a library large enough to have the same file in different places (because my filesystem is a mess).

**Comment by flyingrub** [comment]
The sync process doesn't import the music twice, even if you rerun the sync
after it finished (perhaps, if one is already running, it could do that).
You can install data browser from the store to check if the track have the
same file id.
